docs/examples/openiotsdk_examples.md

```diff
@@ -24,81 +24,61 @@ Matter Project. Please read this
 
     ```
-    docker network create -d macvlan --subnet=192.168.1.0/24 --gateway=192.168.1.1 --ipv6 --subnet=fd12:41:e237:6905::/64 --gateway=fd12:41:e237:6905::11 -o parent=eth0 macvlan`
+    ${MATTER_ROOT}/scripts/setup/openiotsdk/network_setup.sh down
     ```
 
-    The next step is to run Docker container and attach the macvlan network to
-    it using the `--network` option. We can also assign the IP address to our
-    container with `--ip`. Be sure to specify an IP that is not within your DHCP
-    IP range to avoid instances of an IP conflict.
-
-    It is also recommended to add IPv6 traffic forwarding configuration with
-    `--sysctl` option.
+    Use `--help` to get more information about the script options.
 
-    The Vscode devcontainer users should edit the
-    `.devcontainer/devcontainer.json` file and add run options.
+-   **connect_if.sh** - script that connect specific network interfaces with the
+    default route interface. Create a bridge and link all interfaces to it. Keep
+    the default route of network traffic.
 
     Example:
 
     ```
-    ...
-    "runArgs": [
-        ...
-        "--network=macvlan",
-        "--ip=192.168.1.110",
-        "--sysctl",
-        "net.ipv6.conf.all.disable_ipv6=0 net.ipv4.conf.all.forwarding=1 net.ipv6.conf.all.forwarding=1",
-        ...
-    ],
-    ...
+    ${MATTER_ROOT}/scripts/setup/openiotsdk/connect_if.sh ARMhveth
     ```
 
-    If you launch the Docker container directly from CLI, use the above
-    arguments with `docker run` command.
+    Use `--help` to get more information about the script options.
 
-    To set up proper network interfaces inside Docker container use the designed
-    script `scripts/setup/openiotsdk/network_setup.sh`. Select the main Ethernet
-    interface that TAP device should be linked with `--network` option (the
-    default value is **eth0**). To keep Internet access use the DHCP client
-    `--dhcp`.
+After setup Open IoT SDK network environment the user will be able to run Matter
+examples on FVP in an isolated network namespace in TAP device mode.
 
-    To enable Open IoT SDK networking environment:
+To execute the command given as an argument in a specific network namespace use
+the helper script `scripts/run_in_ns.sh`.
 
-    ```
-    ${MATTER_ROOT}/scripts/setup/openiotsdk/network_setup.sh --network eth0 --dhcp up
-    ```
+Example:
 
-    To disable Open IoT SDK networking environment:
-
-    ```
-    ${MATTER_ROOT}/scripts/setup/openiotsdk/network_setup.sh --network eth0 --dhcp down
-    ```
+```
+${MATTER_ROOT}/scripts/run_in_ns.sh ARMns /bin/bash
+```
 
-    Use `--help` to get more information about the script options.
+Use `--help` to get more information about the script options.
 
-    **NOTE**
+**NOTE**
 
-    This option is limited to a Docker running on a Linux host as creating
-    macvlan interface is not properly supported on other systems.
+For Docker environment users it's recommended to use the default Docker bridge
```

Comment:
I don't know what that means, what is the default docker bridge and how do I use it?